Packs/MicrosoftWindowsEvents/ModelingRules/MicrosoftWindowsEvents_1_3/MicrosoftWindowsEvents_1_3.xif

```diff
@@ -32,7 +37,7 @@ filter channel in ("System", "Application") or provider_name in("Microsoft-Windo
         message_task = arrayindex(regextract(message , "task \"([^\"]+)\""), 0),
         new_value =  event_data -> ["New Value"],
         old_value = event_data -> ["Old Value"], 
-        file_path = coalesce(arrayindex(regextract(message , "Path:\s*([^\n]+)\n"), 0), arrayindex(regextract(message , "Path:\s*([^,]*?)(?:\s{2}|,|$)"), 0)),
+        file_path = coalesce(arrayindex(regextract(message , "Path:\s*([^\n]+)\n"), 0), arrayindex(regextract(message , "Path:\s*([^,]*?)(?:\s{2}|,|$)"), 0), event_data -> Path),
```

Comment:
The order here for the **coalesce** is what you intended?
Just saw in previous fields that the sugar extraction was first.